### DISCUSSION TITLE
Failing to get a human-readable `schedule`

### DISCUSSION THREAD

**Discussion by jamesbraza** [renovatebot/renovate#32960] [Request Help] [answered]
### How are you running Renovate?

A Mend.io-hosted app

### Please tell us more about your question or problem

I am trying to get Renovate to run on the first Monday of the month between 2 and 3 am. I have tried the following `schedule` in JSON5:

- `schedule: ["at 2am on the first monday of the month"]`
- `schedule: ["after 2am and before 3am on the first monday of the month"]`
- `schedule: ["on the first monday of the month after 2am and before 3am"]` 

However, none of them will pass `renovate-config-validator` invoked via https://github.com/suzuki-shunsuke/github-action-renovate-config-validator.

### Logs (if relevant)

renovate-config-validator --strict  .github/renovate.json5

 INFO: Validating .github/renovate.json5
ERROR: Found errors in configuration
       "file": ".github/renovate.json5",
       "errors": [
         {
           "topic": "Configuration Error",
           "message": "Invalid schedule: `Invalid schedule: Failed to parse \"on the first monday of the month after 2am and before 3am\"`"
         }
       ]

**Comment by RahulGautamSingh** [comment]
We have deprecated the use of later syntax for scheduling, please use cron syntax

`* 2 * * 1#1` (for your case)

**Reply by jamesbraza** [reply]
Did you mean to put the `1#1`, or was that a typo?

And I am thinking `"on the first monday of the month after 2am and before 3am"` would be `schedule: ["* 2 1-7 * 1"]`:

- `*`: every minute
- `2`: during 2nd hour
- `1-7`: during first week of the month
- `*`: every month
- `1`: during 1st day of the week (Monday)

**Reply by RahulGautamSingh** [reply]
https://www.site24x7.com/tools/crontab/cron-generator.html
You can confirm the description for the cron from here

**Reply by jamesbraza** [reply, marked as answer]
Thank you, and yes it was right:

<img width="600" alt="screenshot of cron generator" src="https://github.com/user-attachments/assets/88246a65-2476-4c25-848c-8975289b7f13">

To circle back on this:

> `* 2 * * 1#1` (for your case)

I'm gonna assume the `1#1` was a typo then.

Thanks again for your help here!
